static_src/components/app_container.jsx

```diff
@@ -108,6 +111,25 @@ export default class AppContainer extends React.Component {
     );
   }
 
+  get openApp() {
+    const route = RouteStore.getRouteURLForApp(this.state.app);
+    if (route) {
+      return (
+        <div>
+          <Action
+            style="primary"
+            href={ `https://${route}` }
+            label="open app"
+            type="outline"
+          >
+          <span>Open app</span>
+        </Action>
+      </div>
+      );
+    }
+    return null;
```

Comment:
Sorry, one more ask, can you reverse the `if` condition? I prefer the fail-fast because it keeps the indentation flat which improves legibility, and it keeps all the checks at the top of the function.
```javascript
get openApp() {
 const route = //...
  if (!route) {
    return null;
  }

  return <Action />;
}
```